crates/python/Cargo.toml

```diff
@@ -16,7 +16,7 @@ name = "qcs_sdk"
 crate-type = ["cdylib"]
 
 [dependencies]
-qcs = { path = "../lib" }
+qcs = { path = "../lib", version = "0.10.0-rc.21" }
```

Comment:
Can we make sure this version is updated when the library is updated? Alternatively, `version = "*"` work?

Reply:
Was able to remove the version spec entirely after replacing the publish action with a simple call to `cargo publish`: https://github.com/rigetti/qcs-sdk-rust/actions/runs/4835196172/jobs/8617271158